modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -345,6 +394,18 @@ protected Instance testPostProcessInstance_addInstance(Instance instance)
 			_process.getId(), instance);
 	}
 
+	private User _addUser(String firstName) throws Exception {
+		return UserTestUtil.addUser(
+			TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+			RandomTestUtil.randomString(
+				NumericStringRandomizerBumper.INSTANCE,
+				UniqueStringRandomizerBumper.INSTANCE),
+			LocaleUtil.getDefault(), firstName,
+			StringUtil.toLowerCase(RandomTestUtil.randomString()),
+			new long[] {TestPropsValues.getGroupId()},
+			ServiceContextTestUtil.getServiceContext());
+	}
+
```

Comment:
Hey @MarinhoFeliphe, Can we use this overloaded `UserTestUtil.addUser` option? 

```java

public static User addUser(
	String screenName, Locale locale, String firstName, String lastName,
	long[] groupIds)
  throws Exception {
  
  return addUser(
	  TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
	  screenName, locale, firstName, lastName, groupIds,
	  ServiceContextTestUtil.getServiceContext());
}
````
Thx